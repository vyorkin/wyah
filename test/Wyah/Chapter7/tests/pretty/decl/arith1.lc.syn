Decl "npred" (ELam (Var "x") (EOp Sub (EVar (Var "x")) (ELit (LInt 1))))
