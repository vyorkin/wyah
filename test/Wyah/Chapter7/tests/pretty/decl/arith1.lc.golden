let npred = \x -> x - 1;